stable/mysql/values.yaml

```diff
@@ -28,6 +28,11 @@ imageTag: "5.7.14"
 ##
 imagePullPolicy: IfNotPresent
 
+## Optional array of imagePullSecrets containing private registry credentials
+## Ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+imagePullSecrets: []
```

Comment:
Can you comment out `imagePullSecrets`? That's the way it's handled elsewhere. Looking for consistency.

For example,
https://github.com/kubernetes/charts/blob/da7d836a29f30c6f5b469686643a72f64dd983e0/incubator/oauth-proxy/values.yaml#L16-L20

Reply:
Thanks changed